substrate/runtime/src/lib.rs

```diff
@@ -143,6 +149,16 @@ parameter_types! {
     );
 
   pub const MaxAuthorities: u32 = 100;
+
+  pub const AssetConversionPalletId: PalletId = PalletId(*b"DexPalet"); // TODO: this requires exactly 8 byte?
+  pub const LiquidityWithdrawalFee: Permill = Permill::from_percent(1);
+  pub const AllowMultiAssetPools: bool = false;
+}
+
+ord_parameter_types! {
+  // TODO: normally we should do AssetConversionPalletId::get() instead of b"DexPalet"
+  // but that gives a lifetime error for now.
+  pub const AssetConversionOrigin: PublicKey = PublicKey::from(pallet_address(b"DexPalet"));
```

Comment:
Pallet, not Palet.

Reply:
This TODO explains that we should normally use `AssetConversionPalletId` to construct the pallet address and if you have a look https://github.com/serai-dex/serai/pull/363#discussion_r1310780660 there is another TODO there that say it should be exactly 8 byte? The reason is pallet ids are fixed 8 bytes. So it was intentionally named `DexPalet` to make it 8 byte because `DexPallet` is 9 byte.